Dockerfile

```diff
@@ -7,10 +7,16 @@ ARG BASE_IMAGE="ghcr.io/mit-dci/opencbdc-tx-base:latest"
 # Create Base Image
 FROM $IMAGE_VERSION AS base
 
+# make shell run with bash not sh
+SHELL ["/bin/bash", "-c"]
+
 # set non-interactive shell
-ENV DEBIAN_FRONTEND noninteractive
-ENV CMAKE_BUILD_TYPE Release
-ENV BUILD_RELEASE 1
+ENV DEBIAN_FRONTEND=noninteractive
+ENV CMAKE_BUILD_TYPE=Release
+ENV BUILD_RELEASE=1
+
+COPY requirements.txt /opt/tx-processor/requirements.txt
+COPY scripts/activate-venv.sh /opt/tx-processor/scripts/activate-venv.sh
```

Comment:
These two lines should be moved down to be with the other copy-scripts-to-container lines.

Reply:
ACK - moved